Formula/felinks.rb

```diff
@@ -0,0 +1,77 @@
+class Felinks < Formula
+  desc "Text mode browser and Gemini, NNTP, FTP, Gopher, Finger, and BitTorrent client"
+  homepage "https://github.com/rkd77/elinks#readme"
+  url "https://github.com/rkd77/elinks/archive/refs/tags/v0.15.0.tar.gz"
+  sha256 "49e261fb7a6b061b51ca38b7152712caba90c8eaa6996bcf4c00fe2867a7f366"
+  license "GPL-2.0-only"
+  head "https://github.com/rkd77/elinks.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "groff" => :build
+  depends_on "libtool" => :build
+  depends_on "xmlto" => :build
+
+  depends_on "brotli"
+  depends_on "libidn"
+  depends_on "openssl@1.1"
+  depends_on "tre"
+  depends_on "zstd"
+
+  uses_from_macos "bison"
+  uses_from_macos "bzip2"
+  uses_from_macos "expat"
+  uses_from_macos "zlib"
+
+  def install
+    # Inherited from the Elinks formula
+    ENV.deparallelize
+    ENV.delete("LD")
+    ENV.append "CFLAGS", "-Wno-implicit-function-declaration"
+
+    # Change name to felinks to avoid clashing with elinks
+    inreplace "configure.ac", "PACKAGE=elinks", "PACKAGE=felinks"
+    inreplace "configure.ac", "grep elinks", "grep felinks"
+    inreplace "configure.ac", "sysconfdir/elinks", "sysconfdir/felinks"
```

Comment:
We shouldn't change the name upstream gave to software, if it conflicts there is `conflicts_with`

Reply:
Understood - I'll make the change and add a conflicts_with ... I assume there is nothing like "provides: elinks?"

Just FYI, the "elinks" that Brew installs is a dead project -  version 0.11.7 was 13 years ago.  This is the active and maintained version and some Linux distributions have already made felinks their default elinks.

Does brew offer any kind of mechanism to hint to users of elinks that that project is abandoned with no releases this decade and suggest they use felinks instead?

Reply:
If it's not there yet you can add a `deprecated!` call to the existing elinks.